src/PointwiseFunctions/AnalyticData/NewtonianEuler/ShuOsherTube.cpp

```diff
@@ -18,28 +17,28 @@
 #include "Utilities/Math.hpp"
 
 namespace NewtonianEuler::AnalyticData {
-ShuOsherTube::ShuOsherTube(const double initial_position,
+ShuOsherTube::ShuOsherTube(const double jump_position,
                            const double mass_density_l, const double velocity_l,
                            const double pressure_l, const double velocity_r,
                            const double pressure_r, const double epsilon,
                            const double lambda) noexcept
     : mass_density_l_(mass_density_l),
       velocity_l_(velocity_l),
       pressure_l_(pressure_l),
-      jump_position_(initial_position),
+      jump_position_(jump_position),
       epsilon_(epsilon),
       lambda_(lambda),
       velocity_r_(velocity_r),
       pressure_r_(pressure_r) {
   ASSERT(mass_density_l_ > 0.0,
-         "The left mass_density must be greater than 0.0 but is "
+         "The left mass_density must be greater than 0 but is "
              << mass_density_l_);
   ASSERT(pressure_l_ > 0.0,
-         "The left pressure must be greater than 0.0 but is " << pressure_l_);
+         "The left pressure must be greater than 0 but is " << pressure_l_);
   ASSERT(pressure_r_ > 0.0,
-         "The right pressure must be greater than 0.0 but is " << pressure_r_);
-  ASSERT(epsilon_ > -1.0,
-         "The sinusoid amplitude must be greater than -1 but is " << epsilon_);
+         "The right pressure must be greater than 0 but is " << pressure_r_);
+  ASSERT(epsilon_ > 0.0,
+         "The sinusoid amplitude must be greater than 0 but is " << epsilon_);
```

Comment:
Consider adding a new ASSERT for checking epsilon < 1 (else negative densities will occur)